docs/journalist.rst

```diff
@@ -645,6 +645,55 @@ audio, and begin publishing important, high-impact work!
          <getting_the_most_out_of_securedrop>` to read about
          encouraging sources to use SecureDrop.
 
+Deleting submissions and source accounts
+----------------------------------------
+
+As part of routine SecureDrop usage, we recommend that you establish data retention
+practices consistent with your organization's threat model. Regularly deleting
+submissions and source accounts can mitigate risks in the event that your
+SecureDrop servers or a source's account details are compromised.
+
+To delete sources, first select them in the list of all sources in the
+*Journalist Interface*, then click the **Delete** button. You will be
+given a choice to delete all messages and files for the selected sources, or to
+delete the source accounts:
+
+|Delete sources|
+
+If you delete messages and files for a source, the source will continue to appear
+in the list of sources in the *Journalist Interface*, and they will still be able
+to log into the *Source Interface* using their codename. Consider using this
+option if you are not sure that all communication with the source has concluded.
```

Comment:
or as part of regular maintenance to reduce the disk usage on the server as you move those files to offline storage